static/js/templates.js

```diff
@@ -2,6 +2,14 @@ var templates = (function () {
 
 var exports = {};
 
+function trans(str, context) {
+    if (typeof i18n !== 'undefined') {
+        // only translate the string if i18n is loaded
+        return i18n.t(str, context);
+    }
+    return str;
+}
```

Comment:
Yeah, I have only come across this problem in my local environment. The better way would be to make sure that i18n is loaded before rendering any Handlebar template. This was kind of a hack. Anyway, I'll try another approach.